openshift/app.dc.yaml

```diff
@@ -69,6 +69,9 @@ objects:
           containers:
             - name: app
               image: "${IMAGE_REGISTRY}/${NAMESPACE}/${REPO_NAME}-app:latest"
+              securityContext:
+                capabilities:
+                  add: ["NET_BIND_SERVICE"]
```

Comment:
If you can change the add section to be YAML style instead of JSON array style that would be ideal:
```yaml
securityContext:
  capabilities:
    add:
      - NET_BIND_SERVICE
```

Reply:
Done!